src/timings/Timings.php

```diff
@@ -310,9 +310,7 @@ public static function getEventHandlerTimings(string $event, string $handlerName
 	public static function getAsyncTasksProgressUpdateTimings(AsyncTask $task, string $group = self::GROUP_BREAKDOWN) : TimingsHandler{
 		$taskClass = $task::class;
 		if(!isset(self::$asyncTasksProgressUpdate[$taskClass])){
-			if (!isset(self::$schedulerAsync)){
-				self::init();
-			}
+			self::init();
```

Comment:
This will reform an instance of all the other handlers when a new class is encountered, won't it?

Reply:
No. init() has its own guard variable to prevent double initialization.

Reply:
So much for me